### PR TITLE
Better error message

### PR DESCRIPTION
**Pull request by deltakosh**

### CODE REVIEW

**Review by sebavan — APPROVED** [review]

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/15815/merge

Test environment: 
https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15815/merge/index.html

To test a playground add it to the URL, for example:

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15815/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/15815/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/15815/merge
https://gui.babylonjs.com/?snapshot=refs/pull/15815/merge
https://nme.babylonjs.com/?snapshot=refs/pull/15815/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/15815/merge#BCU1XR#0

**Comment by bjsplat** [comment]
WebGL2 visualization test reporter:

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15815/merge/testResults/webgl2playwright/index.html

**Comment by bjsplat** [comment]
Visualization tests for WebGPU (Experimental)

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15815/merge/testResults/webgpuplaywright/index.html

File: packages/tools/nodeEditor/src/graphSystem/connectionPointPortData.ts [hunk 1]
```diff
@@ -163,7 +164,7 @@ export class ConnectionPointPortData implements IPortData {
 
         switch (issue) {
             case NodeMaterialConnectionPointCompatibilityStates.TypeIncompatible:
-                return "Cannot connect two different connection types";
+                return `Cannot connect two different connection types:\nSource is ${NodeMaterialBlockConnectionPointTypes[this.data.type]} and destination is ${NodeMaterialBlockConnectionPointTypes[(targetPort.data as NodeMaterialConnectionPoint).type]}`;
```
[the remark this answers is not in the document]
Comment (deltakosh) [reply packages/tools/nodeEditor/src/graphSystem/connectionPointPortData.ts:167]: Good point! I'll add the accepted types

**Pull request merged**